src/features/class-parsely-support.php

```diff
@@ -60,6 +60,10 @@ public function add_parsely_trending_posts_query( array $posts, array $args ): a
 	 * @return array<int> An array of post IDs.
 	 */
 	public function get_trending_posts( array $args ): array {
+		$parsely = $GLOBALS['parsely'];
```

Comment:
Is this the best way to check this? By accessing the global directly? Curious

Reply:
yeah, it's already instantiated, and that's the way parsely refers to the instance.